### PR TITLE
autoupdate custom plugins

### PR DESCRIPTION
**Pull request by benwaffle**

### CODE REVIEW

**Comment by slavaGanzin** [comment]
:+1: But what is $ZSH/custom?

**Comment by benwaffle** [comment]
`$ZSH/custom` is where other plugins go, e.g. [zsh-syntax-highlighting](https://github.com/zsh-users/zsh-syntax-highlighting/blob/master/INSTALL.md#oh-my-zsh)

File: tools/upgrade.sh [hunk 1]
```diff
@@ -37,3 +37,11 @@ then
 else
   printf "${RED}%s${NORMAL}\n" 'There was an error updating. Try again later?'
 fi
+
+printf "${BLUE}%s${NORMAL}\n" "Updating Custom Plugins"
+cd "$ZSH/custom/plugins"
```
Comment (slavaGanzin) [inline tools/upgrade.sh]: Let's use $ZSH_CUSTOM/plugins.

File: tools/upgrade.sh [hunk 2]
```diff
@@ -37,3 +37,11 @@ then
...
+for dir in ./*; do
```
Comment (slavaGanzin) [inline tools/upgrade.sh]: Let's substitute ```./*``` with ```$ZSH_CUSTOM/plugins/*```
And so ```cd``` would be obsolete

Comment (benwaffle) [inline tools/upgrade.sh]: upgrade.sh does not have access to `$ZSH_CUSTOM`
It's run like this:
```zsh
function upgrade_oh_my_zsh() {
  env ZSH=$ZSH sh $ZSH/tools/upgrade.sh
}
```

**Comment by benwaffle** [comment]
This PR now uses `$ZSH_CUSTOM` for the custom plugins dir

**Comment by benwaffle** [comment]
@slavaGanzin do you have write access to the repo? or do we need @robbyrussell ?

What else is needed to merge this PR?

**Comment by slavaGanzin** [comment]
@benwaffle Ben, I have no access. So this should be merged by maintainers

**Comment by benwaffle** [comment]
superseded by #7539

**Pull request closed without merging**